html/includes/graphs/sensor/voltage.inc.php

```diff
@@ -11,8 +11,8 @@
 $rrd_options .= " DEF:sensor_max=$rrd_filename:sensor:MAX";
 $rrd_options .= " DEF:sensor_min=$rrd_filename:sensor:MIN";
 
-$rrd_options .= ' AREA:sensor_max#c5c5c5';
-$rrd_options .= ' AREA:sensor_min#ffffffff';
+//$rrd_options .= ' AREA:sensor_max#c5c5c5';
```

Comment:
Delete rather than comment.

But, I don't see what the original lines do...  There are no min/max drawn on my voltage graphs...